### PR TITLE
Fix for issue #481

### PR DESCRIPTION
**Pull request by gets0ul**
#481

### CODE REVIEW

File: client/src/components/ProfileCard/helper.js [hunk 1]
```diff
@@ -198,9 +198,27 @@ export async function updateUserSkills(apiClient, { id, skills }) {
...
+          apiClient,
+          skill.skillProviderId,
+          skill.externalId
+        );
+        if (existingSkill && existingSkill.length > 0) {
+          skillId = existingSkill[0].id;
+        }
+      }
+      if (skillId) {
+        url = `${config.API_URL}/users/${id}/skills/${skillId}`;
```
Comment (callmekatootie) [inline client/src/components/ProfileCard/helper.js:219]: @cagdas001 Can you let me know the scenario for this code. If the skill did not exist on the user, then if it is added and then deleted (without leaving the edit profile modal), on save, the net effect is null / void - no changes expected.

If a skill existed on the user, then it should definitely always have an id...

Comment (cagdas001) [reply client/src/components/ProfileCard/helper.js:219]: @callmekatootie This PR is opened by another member. It's not mine

Comment (callmekatootie) [reply client/src/components/ProfileCard/helper.js:219]: Crap... My bad... @gets0ul ^

Comment (gets0ul) [reply client/src/components/ProfileCard/helper.js:219]: @callmekatootie This is actually a separate issue I found when fixing this adding/deleting/adding problem.

In Edit Profile modal, add a new skill and then save. The modal dialog will close after save is done.
Now open the Edit again, delete that newly added skill, then save.

Check the console and network tabs in Developer tools, showing log of 404 error failing to delete skill with undefined id.
`DELETE https://api.topcoder-dev.com/v5/users/bdcb113f-6715-40fd-8dab-14aa01327ae9/skills/undefined 404`

> If a skill existed on the user, then it should definitely always have an id...
But newly added skill won't, unless we hard refresh/reload the page.

Comment (callmekatootie) [reply client/src/components/ProfileCard/helper.js:219]: Makes sense. Good catch

**Pull request merged**